backend/src/src-fossology/src/main/java/com/siemens/sw360/fossology/handler/FossologyFileHandler.java

```diff
@@ -28,10 +28,7 @@
 import org.springframework.stereotype.Component;
 
 import java.io.InputStream;
-import java.util.Collection;
-import java.util.Map;
-import java.util.Objects;
-import java.util.Set;
+import java.util.*;
```

Comment:
(do not change that, just a remark) actually I like identifying individual objects more since the loaded objects are less? - but after years I am not sure about it any more and one must consistently doing this to have this as an effect.